compiler/luci/service/include/luci/Service/CircleShapeInference.h

```diff
@@ -122,7 +122,15 @@ class Algorithm final : public luci::CircleNodeVisitor<loco::TensorShape>
   // loco::TensorShape visit(const luci::CircleRelu0To1 *node) final;
   // loco::TensorShape visit(const luci::CircleRelu6 *node) final;
   // loco::TensorShape visit(const luci::CircleReluN1To1 *node) final;
-  // loco::TensorShape visit(const luci::CircleReshape *node) final;
+
+  /**
+   * @note  CircleReshape has new shape info in two places: 2nd input and attribute.
```

Comment:
Please move the comment to the implementation part, not here :)

And the policy may be changed by this PR..!